src/components/ProBanner.tsx

```diff
@@ -0,0 +1,21 @@
+import { OcLinkexternal2 } from "solid-icons/oc";
+
+import { useGlobalContext } from "../context/Global";
+
+const ProBanner = () => {
+    const { t } = useGlobalContext();
+
+    return (
+        <div class="banner banner-yellow">
+            <a
+                href="https://docs.boltz.exchange/boltz-client/boltz-pro"
```

Comment:
I don't like it saying "use API" but linking to an API client. Imho it should link to https://docs.boltz.exchange/api/pro

Reply:
Fine, I am giving in if you finally get me https://github.com/BoltzExchange/boltz-backend/pull/936 merged